lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -214,6 +246,7 @@ def render_options
           presented_options(recalculate: true).each do |choice, num|
             padding = ' ' * (max_num_length - num.to_s.length)
             message = "  #{num}#{num ? '.' : ' '}#{padding}"
+            message += " #{CHECKBOX_ICON[@chosen[num - 1]]} " if @multiple && num && num > 0
```

Comment:
Can we add colour as well?

Reply:
Not sure what's best. Yellow/Green? On the checkbox only or the whole line? Perhaps make the line's text bold only if selected?

Reply:
I'd say whole line, but not super opinionated on how exactly we style it, just that some kind of styling seems like the dev way. 😛

Reply:
I went for "standard vs bold+cyan", other options looked like a Christmas tree. See screenshot in description.